pom.xml

```diff
@@ -1461,5 +1461,28 @@
         <maven.compiler.release>21</maven.compiler.release>
       </properties>
     </profile>
+    <profile>
+      <id>may-spotless-apply</id>
+      <build>
```

Comment:
@Vlatombe 
Couldn't the profile be automatically applied depending on a property?

```
      <id>may-spotless-apply</id>
      <activation>
        <property>
          <name>!spotless.check.skip</name>
        </property>
      </activation>
      <build>
```

Reply:
IIRC you cannot activate profiles based on properties defined in the project POM, only on user properties.